CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Zebra 1.0.0-beta.6](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.6) - 2022-03-28
 
-Zebra's latest beta brings the RPC server support and part of needed API calls to become a **lightwalletd** back end.
+Zebra's latest beta validates all the NU5 consensus rules.
```

Comment:
That's not strictly true yet, it is not up to date with the ZIP-244 changes, which are happening in the `nu5` branch. I think it's safer to remove this phrase in this release, and in the next one (which is the one to use in testnet reactivation) we can include it.

Reply:
https://github.com/ZcashFoundation/zebra/pull/3966/commits/f9f62af2ced537089f89438eabd040e492993a97